cuda_core/cuda/core/experimental/_linker.py

```diff
@@ -37,28 +39,35 @@ def _decide_nvjitlink_or_driver() -> bool:
 
     _driver_ver = handle_return(driver.cuDriverGetVersion())
     _driver_ver = (_driver_ver // 1000, (_driver_ver % 1000) // 10)
+
+    def libname():
+        return "nvJitLink*.dll" if sys.platform == "win32" else "libnvJitLink.so*"
```

Comment:
nit: this does not have to be a callable, it can just be `libname = A if condition else B`.

Reply:
Resolved via commit 6bf421ed0bd3a2460b62c7252d54365bfe7c8fa0